src/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const MIN_CODE_ANALYZER_VERSION_REQUIRED = '5.0.0-beta.0'
+export const MIN_CODE_ANALYZER_VERSION_REQUIRED = '5.0.0'
```

Comment:
I made a call to update this version, but this could break existing implementations. Is this the right approach?